hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestDecommissioningStatus.java

```diff
@@ -307,6 +323,59 @@ public void testDecommissionStatus() throws Exception {
     AdminStatesBaseTest.cleanupFile(fileSys, file2);
   }
 
+  protected void verifyInitialState(FSNamesystem fsn, DatanodeManager dm)
```

Comment:
Would you document why do we need to check it?

Reply:
Sure thing, let me do it right away.